includes/revisiondelete/RevisionDelete.php

```diff
@@ -26,19 +26,38 @@ public static function getRelationType() {
 	public function doQuery( $db ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)

[project-id]: wikia:app